### PR TITLE
Login page design (Resolves #15)

### LINKED ISSUE #15: Login page design
Login page should be similar to mockups in design and responsive to mobile devices.

### PR DESCRIPTION
**Pull request by kundanvishen**
This PR Resolves #15 
Changes:
**Desktop**:
image
**Tablets**
image

**Mobile Devices**
image

### CODE REVIEW

**Review by adityad85 — DISMISSED** [review]

**Review by andrewlinfoot — CHANGES_REQUESTED** [review]
loving the screenshots in the PR. you should make more components to abstract away your classes. this is super useful for keeping a consistent look across the app. watch this video and refactor: https://www.youtube.com/watch?v=gNeavlJ7lNY

File: src/components/LoginPage/LoginPage.js [hunk 1]
```diff
@@ -19,78 +19,122 @@ class LoginPage extends React.Component {
...
+                                <span>
+                                  {meta.error}
+                                </span>
+                                )
+                              }
+                            </div>
+                          )}
+                        </Field>
+                        <div className="row mx-auto">
+                          <button
```
Comment (andrewlinfoot) [inline src/components/LoginPage/LoginPage.js]: should be a button component

File: src/components/LoginPage/LoginPage.js [hunk 2]
```diff
@@ -19,78 +19,122 @@ class LoginPage extends React.Component {
...
+                        <Field name="password" validate={validations.composeValidators(validations.required, validations.minValue(8))}>
+                          {({ input, meta }) => (
+                            <div>
+                              <input
+                                {...input}
+                                type="password"
+                                placeholder="Enter password"
+                                className="form-control mt-2 mb-2"
+                              />
+                              {
```
Comment (andrewlinfoot) [inline src/components/LoginPage/LoginPage.js]: would make an `<Error meta={meta} />` component

File: src/components/LoginPage/LoginPage.js [hunk 3]
```diff
@@ -19,78 +19,122 @@ class LoginPage extends React.Component {
...
+                        </Field>
+                      </div>
+                      <div>
+                        <span>
+                          Password
+                        </span>
+                        <Field name="password" validate={validations.composeValidators(validations.required, validations.minValue(8))}>
+                          {({ input, meta }) => (
+                            <div>
+                              <input
```
Comment (andrewlinfoot) [inline src/components/LoginPage/LoginPage.js]: would make an `Input` component that adds the className "form-control"

File: src/components/LoginPage/LoginPage.js (hunk 1 above)
Comment (kundanvishen) [reply src/components/LoginPage/LoginPage.js]: Thanks! Will make the changes 👍

**Comment by kundanvishen** [comment]
@andrewlinfoot advised changes have been made. Please review.

File: src/components/RegisterPage/RegisterPage.js [hunk 4]
```diff
@@ -42,64 +42,64 @@ class RegisterPage extends React.Component {
 
             <form name="form" onSubmit={handleSubmit}>
               <div>
-                name
+                  name
                 <Field name="name" validate={validations.required}>
                   {({ input, meta }) => (
                     <div>
                       <input {...input} type="text" placeholder="Enter your name" />
                       {meta.error && meta.touched && (
```
Comment (andrewlinfoot) [inline src/components/RegisterPage/RegisterPage.js]: Use the FormError component here

File: src/components/RegisterPage/RegisterPage.js [hunk 5]
```diff
@@ -42,64 +42,64 @@ class RegisterPage extends React.Component {
...
                 email
                 <Field name="email" validate={validations.composeValidators(validations.required, validations.emailFormat)}>
                   {({ input, meta }) => (
                     <div>
                       <input {...input} type="email" placeholder="Enter your email" />
                       {meta.error && meta.touched && (
-                      <span>
-                        {meta.error}
-                      </span>
+                        <span>
```
Comment (andrewlinfoot) [inline src/components/RegisterPage/RegisterPage.js]: And here

File: src/components/RegisterPage/RegisterPage.js [hunk 6]
```diff
@@ -42,64 +42,64 @@ class RegisterPage extends React.Component {
...
+                  email
                 <Field name="password" validate={validations.composeValidators(validations.required, validations.minValue(8))}>
                   {({ input, meta }) => (
                     <div>
                       <input {...input} type="password" placeholder="Enter your Password" />
                       {meta.error && meta.touched && (
-                      <span>
-                        {meta.error}
-                      </span>
+                        <span>
```
Comment (andrewlinfoot) [inline src/components/RegisterPage/RegisterPage.js]: And here

**Review by andrewlinfoot — CHANGES_REQUESTED** [review]

**Comment by kundanvishen** [comment]
@andrewlinfoot I should not have added RegisterPage component to this PR 👎 
Everything related to RegisterPage would be handled in RegsiterPage Design PR.
Would a blank RegisterPage component work for now?

**Comment by andrewlinfoot** [comment]
Awh got it, LGTM then

**Review by andrewlinfoot — APPROVED** [review]

**Pull request merged**